### PR TITLE
259799 show identity federation info

### PR DESCRIPTION
**Pull request by nilsgstrabo**

### CODE REVIEW

**Review by Pespiri — CHANGES_REQUESTED** [review]

File: src/components/page-active-component/active-component-overview.tsx [hunk 1]
```diff
@@ -31,23 +31,21 @@ import { routeWithParams } from '../../utils/string';
 import { RootState } from '../../init/store';
 
 import './style.css';
+import { EnvironmentModel } from '../../models/environment';
```
[the remark this answers is not in the document]
Comment (nilsgstrabo) [reply src/components/page-active-component/active-component-overview.tsx]: fixed

File: src/components/page-active-component/active-component-overview.tsx [hunk 2]
```diff
@@ -31,23 +31,21 @@ import { routeWithParams } from '../../utils/string';
...
 
 interface ActiveComponentOverviewState {
   appAlias?: {
     componentName: string;
     environmentName: string;
     url: string;
   };
+  environment?: EnvironmentModel;
```
[the remark this answers is not in the document]
Comment (nilsgstrabo) [reply src/components/page-active-component/active-component-overview.tsx:41]: fixed

File: src/components/page-active-component/overview.tsx [hunk 3]
```diff
@@ -96,4 +104,5 @@ Overview.propTypes = {
   }),
   envName: PropTypes.string.isRequired,
   component: PropTypes.shape(ComponentModelValidationMap).isRequired,
+  deployment: PropTypes.shape(DeploymentModelValidationMap).isRequired,
```
[the remark this answers is not in the document]
Comment (nilsgstrabo) [reply src/components/page-active-component/overview.tsx:107]: fixed

File: src/components/page-active-job-component/active-job-component-overview.tsx [hunk 4]
```diff
@@ -42,9 +42,11 @@ import {
 import { getEnvsUrl } from '../../utils/routing';
 import { routeWithParams } from '../../utils/string';
 import { ComponentReplicaList } from '../page-active-component/component-replica-list';
+import { EnvironmentModel } from '../../models/environment';
 
 interface ActiveScheduledJobOverviewState {
   component?: ComponentModel;
+  environment?: EnvironmentModel;
```
[the remark this answers is not in the document]
Comment (nilsgstrabo) [reply src/components/page-active-job-component/active-job-component-overview.tsx:45]: fixed

File: src/components/page-active-job-component/overview.js [hunk 5]
```diff
@@ -27,7 +30,8 @@ export const Overview = ({ component }) => (
 );
 
 Overview.propTypes = {
-  component: PropTypes.shape(ComponentModelValidationMap),
+  component: PropTypes.shape(ComponentModelValidationMap).isRequired,
+  deployment: PropTypes.shape(DeploymentModelValidationMap).isRequired,
```
[the remark this answers is not in the document]
Comment (nilsgstrabo) [reply src/components/page-active-job-component/overview.js:34]: Deployment is required. A component cannot exist without a deployment.

File: src/models/component/index.ts [hunk 6]
```diff
@@ -28,6 +29,7 @@ export interface ComponentModel {
   variables?: Record<string, string>;
   replicaList?: Array<ReplicaSummaryNormalizedModel>;
   horizontalScalingSummary?: HorizontalScalingSummaryModel;
+  identity?: IdentityModel;
```
[the remark this answers is not in the document]
Comment (nilsgstrabo) [reply src/models/component/index.ts:32]: fixed

**Review by Pespiri — APPROVED** [review]

**Pull request merged**